CONTRIBUTING.md

```diff
@@ -358,6 +358,12 @@ subsets of tests.
 See the [bats usage documentation](https://bats-core.readthedocs.io/en/stable/usage.html)
 for details.
 
+## Man Pages
+
+Unreleased changes to `man` pages are available from the `nix develop` shell but
+you will need to restart the shell or call `direnv reload` to pick up new
+changes.
```

Comment:
````suggestion
## Man Pages

Unreleased changes to `man` pages are available from the `nix develop` shell but
you will need to restart the shell or call `direnv reload` to pick up new
changes.
To iteratively edit manpages the files in the `flox-manpages` can also be passed to `man` explicitly:

```shell
$ nix build .#flox-manpages
$ man -i ./result/share/man/man1/flox.1
```



````

Reply:
As discussed, I'm going to proceed without this for now so that we only document one consistent way of doing it.